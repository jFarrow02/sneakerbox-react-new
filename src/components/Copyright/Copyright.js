import React from 'react';
import './Copyright.scss';

/**
 * The Copyright component renders copyright information for the application.
 */
const Copyright = ()=>{

    const now = new Date(Date.now());
    return(
        <div className='Copyright'>
            <p>Copyright&copy; { now.getUTCFullYear() } 908 Productions Inc. All rights reserved.</p>
        </div>
    );
}

export default Copyright;