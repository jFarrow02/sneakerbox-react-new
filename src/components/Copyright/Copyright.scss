@import '../../styles/variables.scss';

@font-face{
    font-family: 'Saria';
    src: url($sairaStencil);
}

.Copyright{
    font-family: 'Saria';
}