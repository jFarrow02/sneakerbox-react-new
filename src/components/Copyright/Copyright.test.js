import React from 'react';
import Copyright from './Copyright';
import renderer from 'react-test-renderer';

it('should match a snapshot', ()=>{
    const tree = renderer.create(<Copyright/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
})