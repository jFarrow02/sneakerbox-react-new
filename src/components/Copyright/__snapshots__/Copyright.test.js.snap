// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match a snapshot 1`] = `
<div
  className="Copyright"
>
  <p>
    CopyrightÂ© 
    2019
     908 Productions Inc. All rights reserved.
  </p>
</div>
`;
